src/main/javascript/crypto/src/Authenticator.ts

```diff
@@ -197,9 +196,13 @@ export class Authenticator {
 
         let pok = crypto.computeAttestationProof(secret, nonce);
         let attRequest = AttestationRequest.fromData(crypto.getType(type), pok);
-        let attest = new Eip712AttestationRequest(userKey);
-        await attest.addData(attestorDomain, 20*1000, receiverId, attRequest);
-        let attestJson = attest.getJsonEncoding();
+        let attestationRequest = new Eip712AttestationRequest(userKey);
+        await attestationRequest.addData(attestorDomain, undefined, receiverId, attRequest);
```

Comment:
Why `undefined` validity? How exactly is that encoded? Shouldn't it be default validity?

Reply:
when function receive "undefined" then it use default value